docs/python/testing.md

```diff
@@ -313,6 +313,32 @@ Below are all the supported commands for testing with the Python extension in VS
 |  **Testing: Focus on Test Explorer View** | Open the Test Explorer view. Similar to **Testing: Focus on Python View** on versions prior to 2021.9.
 |  **Test: Stop Refreshing Tests** | Cancel test discovery. |
 
+## Django unit tests
+
+The Python extension also offers support for discovering and running Django unit tests! You can get your Django tests discovered with only a few additional setup steps:
+
+
+1. Set `"python.testing.unittestEnabled": true,` in your `settings.json` [file](/docs/getstarted/settings.md#settingsjson).
+2. Add `MANAGE_PY_PATH` as an environment variable:
+    1. Create a `.env` file at the root of your project.
+    2. Add `MANAGE_PY_PATH='<path-to-manage.py>'` to the `.env` file, replacing `<path-to-manage.py>` with the path to your application's `manage.py` file.
+        > **Tip**: you can copy the path by right clicking on the file in the Explorer view and selecting **Copy Path**.
+    4. Add Django test arguments to `"python.testing.unittestArgs": []` in the `settings.json` [file](/docs/getstarted/settings.md#settingsjson) as needed, and remove any arguments that are not compatible with Django.
```

Comment:
```suggestion
    3. Add Django test arguments to `"python.testing.unittestArgs": []` in the `settings.json` [file](/docs/getstarted/settings.md#settingsjson) as needed, and remove any arguments that are not compatible with Django.
```

Reply:
Should this step be nested or represented top-level #3

Reply:
It should probably be top-level #3 because it is not part of setting that environment variable.